src/Umbraco.Core/Constants-WebhookEvents.cs

```diff
@@ -4,29 +4,83 @@ public static partial class Constants
 {
     public static class WebhookEvents
     {
-        /// <summary>
-        ///     Webhook event name for content publish.
-        /// </summary>
-        public const string ContentPublish = "ContentPublish";
-
-        /// <summary>
-        ///     Webhook event name for content delete.
-        /// </summary>
-        public const string ContentDelete = "ContentDelete";
-
-        /// <summary>
-        ///     Webhook event name for content unpublish.
-        /// </summary>
-        public const string ContentUnpublish = "ContentUnpublish";
-
-        /// <summary>
-        ///     Webhook event name for media delete.
-        /// </summary>
-        public const string MediaDelete = "MediaDelete";
-
-        /// <summary>
-        ///     Webhook event name for media save.
-        /// </summary>
-        public const string MediaSave = "MediaSave";
+        public static class Names
```

Comment:
Since all webhook events now have an alias, shouldn't the name be retrieved from the localization service (and thereby also easily overridable using `lang.user.xml` files)?

I can see some users might want to provide a nicer description of what the event actually does, especially if you have culture/segment variants (to be explicit when it fires) or more advanced custom events... If the name already gets retrieved from the translations, it would be trivial to do the same for descriptions (similarly to actions, see the `actions`, `actionCategories` and `actionDescriptions` translation areas).